image_transport/include/image_transport/image_transport.hpp

```diff
@@ -54,7 +54,7 @@ Publisher create_publisher(
   rclcpp::Node * node,
   const std::string & base_topic,
   rmw_qos_profile_t custom_qos = rmw_qos_profile_default,
-  rclcpp::PublisherOptions options = rclcpp::PublisherOptions());
+  std::optional<rclcpp::PublisherOptions> options = {});
```

Comment:
We should probably include the `<optional>` header in each file where `std::optional` is used.